Makefile-rpm.mk

```diff
@@ -11,19 +11,7 @@ GIT_INFO := $(GIT_NUM_COMMITS).g$(GIT_SHORT)
 BUILD_DEFINES := --define "%relval $(GIT_INFO)"
 RPM_BUILD_OPTIONS := $(BUILD_DEFINES)
 
-%.$(SRC_EXT): %
-	rm -f $@
-	gzip $<
-
-$(NAME)-$(VERSION).tar:
-	git archive --format tar --prefix $(NAME)-$(VERSION)/ -o $@ HEAD
-
-v$(VERSION).tar: $(NAME)-$(VERSION).tar
-	mv $< $@
-
 include packaging/Makefile_packaging.mk
 
-show_git_metadata:
-	@echo $(GIT_SHA1):$(GIT_SHORT):$(GIT_NUM_COMMITS)
-
-.PHONY: show_git_metadata
+$(NAME)-$(VERSION).tar.$(SRC_EXT):
```

Comment:
You can't use `$(SRC_EXT)` here and then hard code the `gzip` on the line below.  That would allow somebody to change just `SRC_EXT` above and then create an invalid archive.

At least if the rule to make whatever target is the result of just changing `SRC_EXT` doesn't exist, it's pretty obvious what the fix is.  Trying to figure out why an archive won't unpack at some later stage would be more of a pain to figure out.

Reply:
Changed to be .gz on the next commit pushed.